src/web/browser/coreVitals/coreVitals.ts

```diff
@@ -0,0 +1,74 @@
+import { getCLS, getFID, getLCP, getFCP, getTTFB } from 'web-vitals';
+
+// Stops ts assuming json values are null
+const temp: any = null;
+
+const jsonData = {
+	page_view_id: '',
+	received_timestamp: `${new Date().toISOString()}`,
+	id: 'prod',
+	received_date: new Date().toISOString().slice(0, 10),
+	fid: temp,
+	cls: temp,
+	lcp: temp,
+	fcp: temp,
+	ttfb: temp,
+};
+
+export const coreVitals = (): void => {
+	type CoreVitalsArgs = {
+		name: string;
+		value: number;
+	};
+
+	const getValue = ({ name, value }: CoreVitalsArgs): void => {
+		switch (name) {
+			case 'FCP':
+				jsonData.fcp = parseFloat(value.toFixed(6));
```

Comment:
What about `Math.round(value * 100000) / 100000`? I like how this is just math and there's no type conversion needed.

Reply:
The problem with Math.round is we loose the decimals which some metrics need.

Reply:
Same same
![Screenshot 2021-02-16 at 10 37 27](https://user-images.githubusercontent.com/1336821/108051595-fc6bae80-7042-11eb-8ac7-886faf021301.jpg)

Reply:
Fair enough 👍